src/dapp/walletProvider.ts

```diff
@@ -94,7 +96,7 @@ export class WalletProvider implements IDappProvider {
     /**
      * Fetches the login hook url and redirects the client to the wallet login.
      */
-    async login(options?: { callbackUrl?: string }): Promise<string> {
+    async login(options?: { callbackUrl?: string; token?: string }): Promise<string> {
```

Comment:
Line 141 will overwrite any value provided to this argument here. Is this intentional?

Reply:
This change was displayed due to more commits being made on the development branch and it's not something I wrote.
Basically, I had to do a merge from development first.